devstack/lib/nfp

```diff
@@ -10,10 +9,18 @@ NEUTRON_CONF=$NEUTRON_CONF_DIR/neutron.conf
 NFP_CONF_DIR=/etc/nfp
 echo "TOP-DIR-NFP : $PWD"
 
+function create_nfp_image {
+    TOP_DIR=$TOP_DIR
+    sudo python $DISK_IMAGE_DIR/diskimage-create/disk_image_create.py $DISK_IMAGE_DIR/diskimage-create/conf.json
+    BUILT_IMAGE_PATH=$(cat /tmp/nfp_image_path)
+    upload_image file://$BUILT_IMAGE_PATH
+
+    openstack --os-cloud=devstack-admin flavor create --ram 512 --disk 3 --vcpus 1 m1.nfp-tiny
+}
+
 function prepare_nfp_image_builder {
-    #setup_develop $NFPSERVICE_DIR
-    sudo -H -E pip install -r $DISK_IMAGE_DIR/diskimage-create/requirements.txt
-    sudo apt-get install -y --force-yes qemu-utils
+    setup_develop $NFPSERVICE_DIR
+    sudo -H -E pip install -r $NFPSERVICE_DIR/diskimage-create/requirements.txt
```

Comment:
I have been asked to remove everything related to base mode from here and
that base mode integration can be carried out later.

Regards,
Deepak

On Wed, May 11, 2016 at 2:08 PM, Rajendra Machani <notifications@github.com>
wrote:

> In devstack/lib/nfp
> <https://github.com/oneconvergence/group-based-policy/pull/287#discussion_r62810499>
> :
>
> >  function prepare_nfp_image_builder {
> > -    #setup_develop $NFPSERVICE_DIR
> > -    sudo -H -E pip install -r $DISK_IMAGE_DIR/diskimage-create/requirements.txt
> > -    sudo apt-get install -y --force-yes qemu-utils
> > +    setup_develop $NFPSERVICE_DIR
> > +    sudo -H -E pip install -r $NFPSERVICE_DIR/diskimage-create/requirements.txt
>
> Why setup_develop is invoked here?
> Why qemu-utils installation is removed. ?
>
> Lets make sure anything related to base mode is not missed. And base mode
> installation is not broken.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/oneconvergence/group-based-policy/pull/287/files/cd0ed125a8a81f96304dffee3ed2b257cbcc6f07#r62810499>
>